Gems/PhysX/Code/Source/Utils.cpp

```diff
@@ -1562,6 +1582,20 @@ namespace PhysX
 
             return configuration;
         }
+
+        void SetMaterialsFromHeightfieldProvider(const AZ::EntityId& heightfieldProviderId, Physics::MaterialSelection& materialSelection)
```

Comment:
yeah just trying to be consistent with the rest of the code, but looks like we have 1000s of cases passing by value and via const-ref roughly equally. 
since it's a class, first thought is always pass by const-ref..